tensorflow_datasets/core/dataset_info.py

```diff
@@ -41,13 +41,16 @@
 import posixpath
 import pprint
 import tempfile
+import logging
```

Comment:
I would use `import logging as py_logging`, and keep the `from absl import logging`. Otherwise, there is a risk that people uses the wrong `logging.info(`